support/ci/deploy_mac_launcher.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -e
+
+# fail fast if we aren't on the desired branch or if this is a pull request
+if  [[ "${TRAVIS_BRANCH}" != "$(cat VERSION)" && ("${TRAVIS_PULL_REQUEST}" != "false" || "${TRAVIS_BRANCH}" != "master") ]]; then
+    echo "We only publish on successful builds of master."
+    exit 0
+fi
+
+BINTRAY_REPO=unstable
+if [ "$(cat VERSION)" == "$TRAVIS_TAG" ]; then
+  BINTRAY_REPO=stable
+fi
+
+# kick off the mac unstable build
+echo "Kicking off the $BINTRAY_REPO mac build"
+var_file=/tmp/our-awesome-vars
+mac_builder=admin@74.80.245.236
+
+# first update the copy of the habitat code stored on the mac server to the latest
+ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null \
```

Comment:
Extracting all these repeated SSH parameters into a variable would make these command read much easier... you can just see what they are one time, and then know that they're always being used (and don't have to worry that one command might have slightly different values, whether by accident or by intention).